site/pt-br/community/contribute/docs_ref.md

```diff
@@ -0,0 +1,207 @@
+# Contribua com a documentação do TensorFlow
+
+<a id="doctest"></a>
+
+## Docstrings testáveis
+
+O TensorFlow usa [DocTest](https://docs.python.org/3/library/doctest.html) para testar trechos de código em docstrings Python. O trecho deve ser um código Python executável. Para ativar o teste, acrescente a linha com `>>>` (três sinais de maior). Por exemplo, aqui está um trecho da função `tf.concat` do arquivo-fonte [array_ops.py](https://www.tensorflow.org/code/tensorflow/python/ops/array_ops.py):
+
+```
+<code>def concat(values, axis, name="concat"):
+  """Concatenates tensors along one dimension.
+  ...
+
+  >>> t1 = [[1, 2, 3], [4, 5, 6]]
+  >>> t2 = [[7, 8, 9], [10, 11, 12]]
+  >>> concat([t1, t2], 0)
+  <tf.Tensor: shape=(4, 3), dtype=int32, numpy=
+  array([[ 1,  2,  3],
+         [ 4,  5,  6],
+         [ 7,  8,  9],
+         [10, 11, 12]], dtype=int32)>
+
+  <... more description or code snippets ...>
+
+  Args:
+    values: A list of `tf.Tensor` objects or a single `tf.Tensor`.
+    axis: 0-D `int32` `Tensor`.  Dimension along which to concatenate. Must be
+      in the range `[-rank(values), rank(values))`. As in Python, indexing for
+      axis is 0-based. Positive axis in the rage of `[0, rank(values))` refers
+      to `axis`-th dimension. And negative axis refers to `axis +
+      rank(values)`-th dimension.
+    name: A name for the operation (optional).
+
+    Returns:
+      A `tf.Tensor` resulting from concatenation of the input tensors.
+  """
+
+  <code here>
+```
+
+Note: TensorFlow DocTest uses TensorFlow 2 and Python 3.
```

Comment:
It looks like this file is untranslated, except for the first sentence.

Reply:
@pcoet So here's what happened: there is this tag `<code here>` inside the code block, which Gitlocalize recognized as an unclosed `<code>` tag, and subsequently could not parse the file correctly. I have removed this file for now from this PR, so it can be merged. Will need to take a look if we can somehow go around this parsing error, or update the English file.